sdk/digitaltwins/azure-digitaltwins-core/src/samples/java/com/azure/digitaltwins/core/RelationshipsSyncSamples.java

```diff
@@ -0,0 +1,176 @@
+
+        String buildingModelPayload = SamplesConstants.TEMPORARY_MODEL_WITH_RELATIONSHIP_PAYLOAD
+            .replace(SamplesConstants.MODEL_ID, sampleBuildingModelId)
+            .replace(SamplesConstants.MODEL_DISPLAY_NAME, "Building")
+            .replace(SamplesConstants.RELATIONSHIP_NAME, "contains")
+            .replace(SamplesConstants.RELATIONSHIP_TARGET_MODEL_ID, sampleFloorModelId);
+
+        String floorModelPayload = SamplesConstants.TEMPORARY_MODEL_WITH_RELATIONSHIP_PAYLOAD
+            .replace(SamplesConstants.MODEL_ID, sampleFloorModelId)
+            .replace(SamplesConstants.MODEL_DISPLAY_NAME, "Floor")
+            .replace(SamplesConstants.RELATIONSHIP_NAME, "containedIn")
+            .replace(SamplesConstants.RELATIONSHIP_TARGET_MODEL_ID, sampleBuildingModelId);
+
+        List<ModelData> createdModels = client.createModels(new ArrayList<>(Arrays.asList(buildingModelPayload, floorModelPayload)));
+
+        for (ModelData model : createdModels) {
+            ConsoleLogger.print("Created model " + model.getId());
+        }
+
+        // Create a building digital twin
+        BasicDigitalTwin buildingDigitalTwin = new BasicDigitalTwin()
+            .setId(buildingTwinId)
+            .setMetadata(new DigitalTwinMetadata()
+                .setModelId(sampleBuildingModelId));
+
+        client.createDigitalTwin(buildingTwinId, mapper.writeValueAsString(buildingDigitalTwin));
+
+        ConsoleLogger.print("Created twin" + buildingDigitalTwin.getId());
+
+        BasicDigitalTwin floorDigitalTwin = new BasicDigitalTwin()
+            .setId(floorTwinId)
+            .setMetadata(new DigitalTwinMetadata()
+                .setModelId(sampleFloorModelId));
+
+        client.createDigitalTwin(floorTwinId, mapper.writeValueAsString(floorDigitalTwin));
+
+        ConsoleLogger.print("Created twin" + floorDigitalTwin.getId());
+
+        ConsoleLogger.printHeader("Create relationships");
+
+        BasicRelationship buildingFloorRelationshipPayload = new BasicRelationship()
+            .setId(buildingFloorRelationshipId)
+            .setSourceId(buildingTwinId)
+            .setTargetId(floorTwinId)
+            .setName("contains")
+            .setCustomProperties("Prop1", "Prop1 value")
+            .setCustomProperties("Prop2", 6);
+
+        client.createRelationship(buildingTwinId, buildingFloorRelationshipId, mapper.writeValueAsString(buildingFloorRelationshipPayload));
+
+        ConsoleLogger.printSuccess("Created a digital twin relationship "+ buildingFloorRelationshipId + " from twin: " + buildingTwinId + " to twin: " + floorTwinId);
+
+        ConsoleLogger.printHeader("Get Relationship");
+        Response<BasicRelationship> getRelationshipRepsonse = client.getRelationshipWithResponse(
+            buildingTwinId,
+            buildingFloorRelationshipId,
+            BasicRelationship.class,
+            Context.NONE);
+
+        if (getRelationshipRepsonse.getStatusCode() == HttpURLConnection.HTTP_OK) {
```

Comment:
Is this check required? Wouldn't something other than the accepted response code generate an exception?

Reply:
not really a check but a sample code on how to check status codes ... we don't really need it for the sample to function